esphome/components/bl0942/sensor.py

```diff
@@ -1,27 +1,34 @@
 import esphome.codegen as cg
-import esphome.config_validation as cv
 from esphome.components import sensor, uart
+import esphome.config_validation as cv
 from esphome.const import (
+    CONF_ADDRESS,
     CONF_CURRENT,
     CONF_ENERGY,
+    CONF_FREQUENCY,
     CONF_ID,
+    CONF_LINE_FREQUENCY,
     CONF_POWER,
     CONF_VOLTAGE,
-    CONF_FREQUENCY,
     DEVICE_CLASS_CURRENT,
     DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_FREQUENCY,
     DEVICE_CLASS_POWER,
     DEVICE_CLASS_VOLTAGE,
-    DEVICE_CLASS_FREQUENCY,
     STATE_CLASS_MEASUREMENT,
+    STATE_CLASS_TOTAL_INCREASING,
     UNIT_AMPERE,
+    UNIT_HERTZ,
     UNIT_KILOWATT_HOURS,
     UNIT_VOLT,
     UNIT_WATT,
-    UNIT_HERTZ,
-    STATE_CLASS_TOTAL_INCREASING,
 )
 
+LINE_FREQS = {
+    "50HZ": 50,
+    "60HZ": 60,
+}
```

Comment:
This needs to be moved down in this file
```suggestion
LineFrequency = bl0942_ns.enum("LineFrequency")
LINE_FREQS = {
    50: LineFrequency.LINE_FREQUENCY_50HZ,
    60: LineFrequency.LINE_FREQUENCY_60HZ,
}
```